packages/react-web/src/inbox/SignerAdapter.ts

```diff
@@ -0,0 +1,20 @@
+import { WalletEntity } from '@lens-protocol/react';
+
+interface ISigner {
+  getAddress(): Promise<string>;
+  signMessage(message: ArrayLike<number> | string): Promise<string>;
+}
+
+export class SignerAdapter implements ISigner {
```

Comment:
Shouldn't this implement the XMTP interface rather than a local interface?

Reply:
true, I thought they don't export it, but I was wrong